src/NUnitFramework/framework/Internal/Commands/DelegatingTestCommand.cs

```diff
@@ -38,9 +38,9 @@ public abstract class DelegatingTestCommand : TestCommand
 #pragma warning restore IDE1006
 
         /// <summary>
-        /// TODO: Documentation needed for constructor
+        /// Initializes a new instance of the <see cref="DelegatingTestCommand"/> class.
         /// </summary>
-        /// <param name="innerCommand"></param>
+        /// <param name="innerCommand">The inner command.</param>
```

Comment:
If it was me, I'd either make this more descriptive (explaining what "inner command" means) or delete the `<param>`.